en/core-api/bank-transfer.md

```diff
@@ -1,69 +1,62 @@
-
-?> HTTP request by using Basic Authentication method. The username is your Server Key while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by **:** (colon symbol).
+#### Request Details
+| Environment | Method | URL                                        |
+| ----------- | ------ | ------------------------------------------ |
+| Sandbox     | POST   | https://api.sandbox.midtrans.com/v2/charge |
+| Production  | POST   | https://api.midtrans.com/v2/charge         |
 
-## 1. Send Transaction Data to API Charge
-API request should be done from **Merchant’s backend** to acquire `va_number`. There are several components that are required:
+#### Headers
+| Header Name   | Description                                            | Required | Values                |
+| ------------- | ------------------------------------------------------ | -------- | --------------------- |
+| Accept        | The format of the data to be returned.                 | Required | application/json      |
+| Content-Type  | The format of the data to be posted.                   | Required | application/json      |
+| Authorization | The authentication method used to access the resource. | Required | Basic **AUTH_STRING** |
 
-Requirement | Description
---- | ---
-Server Key | Explained on [previous section](/en/midtrans-account/overview.md)
-`order_id` | Transaction order ID, defined from your side
-`gross_amount` | Total amount of transaction, defined from your side
-`payment_type` | Set Bank Transfer payment method
+**AUTH_STRING**: Base64(`ServerKey + :`)<br>Midtrans API validates HTTP request by using Basic Authentication method. The username is your *Server Key* while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by a colon symbol (**:**). For more details, refer to [ API Authorization and Headers](https://docs.midtrans.com/en/technical-reference/api-header).
 
-Charge API request should be done from Merchant's backend.
+?> ***Note***: *Server Key* is required to authenticate the request. For more details, refer to [HTTPS Header](https://api-docs.midtrans.com/#http-s-header).<br>
 
-### Charge API request
-This is example of basic `/charge` API request in Curl, please implement according to your backend language (you can also check our [available language libraries](/en/technical-reference/library-plugin.md)). The example below shows a sample codes to obtain transaction token:
+#### Sample Request and Request Body
+The sample request for *Charge API* is as given below. The request is in CURL but you can implement it according to your backend language. For more details, refer to available [Language Libraries](/en/technical-reference/library-plugin.md#language-library).The example below shows a sample code to obtain transaction token.
```

Comment:
> a sample code to obtain transaction token.

You should fix this, it is not for obtaining transaction token, it is for retrieving the VA number. which customer will use as destination of their bank transfer.